docs/_config.yml

```diff
@@ -1,8 +1,30 @@
 ---
+
+# Site settings
 name: Java Operator SDK
-title: Java Operator SDK 
+title: Java Operator SDK
+description:            Build Kubernetes Operators in Java without hassle
+baseurl:                # Site subpath, e.g. /blog
```

Comment:
I'm not familiar with Jekyll, but wouldn't having the subpath either omitted or set to / make sense?

Reply:
You are right, I omitted it